docs/reference/form_types.rst

```diff
@@ -86,7 +49,7 @@ Now, the property can be edited by setting a type for each type::
 
     final class PageAdmin extends AbstractAdmin
     {
-        protected function configureFormFields(FormMapper $formMapper)
+        protected function configureFormFields(FormMapper $formMapper): void
```

Comment:
Check $formMapper, I think it should be $form

Reply:
Indeed, fixed